### DISCUSSION TITLE
Android Sync - Import Connection .toml Files

### DISCUSSION THREAD

**Discussion by sgr78** [rustdesk/rustdesk#6405] [General]
I have verified,
The android app uses the SAME .toml configuration files like the desktop version
the developers should point the data store of the app to save .toml connection files
TO \sdcard\android\data\com.carriez.flutter_hbb\app_flutter\peers 
INSTEAD of \data\data\com.carriez.flutter_hbb\app_flutter\peers
THUS will enable users to sync desktop connection files from pc to android and vice versa quite easily.

Access to \data\data\com.carriez.flutter_hbb can be granted to user 
A. On unrooted device only within app, if there is an import feature to Import .toml files 
as the app has local access rights to it's own storage space
B. On rooted devices with any File Manager MiX, Root Explorer etc

So PLEASE either -redirect- peers storage to local storage or enable import of .toml files

_Originally posted by @sgr78 in https://github.com/rustdesk/rustdesk/discussions/6387#discussioncomment-7559179_

**Comment by Info-BC** [comment]
Hello, I can't find any peers directory on my android 13 phone (non root, with some connections history in the app), I have android\data\com.carriez.flutter_hbb with only an empty "files" directory.

If I create app_flutter\peers\ and copy all my .toml files, nothing change in the app. Could-you tell me how to find the location of peers directory?

Thanks
